design-guidelines/copy-and-content/date-time-formatting.md

```diff
@@ -0,0 +1,162 @@
+# Date and Time Formatting
+
+Dates and times should be displayed as concisely as possible without preventing the user from seeing the level of detail required to complete their task.
+
+## Referring to dates and times
+
+When writing copy that refers to dates or times, use the same kind of language you would use in written communication.
```

Comment:
What does "when writing copy" mean?

Reply:
When writing any text that appears in the app. I should find a better way to say this 👍

Reply:
"When writing copy for instruction text, descriptions and longer text, use either the approved abbreviation or full format as best for the appropriate context so the text reads clearly. 

[examples you have below]

"For tables, cards, and other components where space is at a premium, default should use the abbreviated format to maximize spacing, efficiency in readability, and consistency across our data tables."